public/app/store/configureStore.ts

```diff
@@ -15,23 +12,13 @@ export function addRootReducer(reducers: any) {
 }
 
 export function configureStore(initialState?: Partial<StoreState>) {
-  const logger = createLogger({
-    predicate: (getState) => {
-      return getState().application.logActions;
-    },
-  });
-
-  const middleware = process.env.NODE_ENV !== 'production' ? [toggleLogActionsMiddleware, logger] : [];
-
-  const reduxDefaultMiddleware = getDefaultMiddleware<StoreState>({
-    thunk: true,
-    serializableCheck: false,
-    immutableCheck: false,
-  } as any);
-
-  const store = reduxConfigureStore<StoreState>({
+  const store = reduxConfigureStore({
     reducer: createRootReducer(),
-    middleware: [...reduxDefaultMiddleware, ...middleware] as [ThunkMiddleware<StoreState>],
+    middleware: (getDefaultMiddleware) =>
+      getDefaultMiddleware({
+        serializableCheck: false,
+        immutableCheck: { ignoredPaths: getPathsToIgnoreMutationAndSerializableCheckOn() },
```

Comment:
yea the immutableCheck is not great for performance reasons.

Reply:
can we try that again updating rtk to [1.5.1](https://github.com/reduxjs/redux-toolkit/releases/tag/v1.5.1) they seem to improved performance?

Reply:
Can we do this after v8 stable?

Reply:
sure. no need to rush with this.